images/common/utils.sh

```
@@ -229,28 +229,27 @@ function openvpn_config {
 	fi
 }
 
-function openvpn_config_checksum {
+openvpn_config_checksum() {
 	export OFILE=$(curl --silent --insecure \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)